### ISSUE TITLE
After fixing #1032 I now have the exclamation mark issue

### ISSUE THREAD

**Issue by Hoiss** [ukanth/afwall#1041]
Hi ukanth,

thanks for fixing #1032, I can use again the afwall+!
I now have a new issue, always showing the exclamation mark next to the connection symbol, regardless if wifi or mobile connection.
I've seen the old issues with the "captive-portal-check", but I assume I should see in the logs if something like this is blocked? I can't find any relevant process there and in the process list there's only a "CaptivePortalLogin" process - if I give it access to the internet the exclamation mark is still there...
I mean...I have access to the internet and could ignore this exclamation mark, but then I wouldn't get if the internet connection is really broken.
Also using NetGuard, I do not have this issue, that's why I want to ask if this is another issue on Android 10 or just a configuration problem? (configuration didn't change, so the export should look the same as in #1032)

**Comment by oushidian** [comment]
I have always had this since I started using afwall. It's because I block the random Google connectivity apps they keep inventing ( I think). When I connect to WiFi it always pops up and said "Problems" and I click don't tell me this again, and I've never had problems. LTE just works.

I don't think it's the portal app as that only gets used when you have to log into a web browser on an open network. There's a Google connectivity app and a metrics app. You can experiment with which one you want to unblock but it just phones home to Google.

**Comment by Hoiss** [comment]
I agree that it should be visible because of a blocked service. But it should be visible in the logs and it isn't.
This is why I ask what's still wrong with afwall.

**Comment by Hoiss** [comment]
I also had this issue with up to date version 3.3.1.
But I just disabled and enabled the log again and now can also see the blocked service leading to the exclamation mark symbol. Even if this behavior is a bit strange, this version seems to to it's job again.
Thanks

**Comment by TheShanMan** [comment]
In addition to enabling captive portal login, I also had to enable "NetworkStack, com.android.server.NetworkPermissionConfig". I'm not sure what all this does, and as far as I'm aware everything else has been working great. But unless I enable this, I get an x on my wifi icon. I'm running LineageOS 17.1 for microG if that matters.
